### PR TITLE
Add docker-compose setup for trying out the applications

### PR DESCRIPTION
**Pull request by rudigiesler**
This docker compose setup should allow people to very easily spin up all the required infrastructure to be able to play around with the USSD applications.

Currently, only the Public USSD line is implemented.

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    environment:
+      - POSTGRES_PASSWORD=postgres
+    restart: always
+
+  identitystore:
+    image: praekeltfoundation/seed-identity-store
+    links:
+      - postgres
+    environment:
+      - IDENTITIES_DATABASE=postgres://postgres:postgres@postgres/postgres
```
Comment (erikh360) [inline docker-compose/docker-compose.yml]: I see all the seed services are connecting to a database called postgres, is this intentional?

**Review by erikh360 — COMMENTED** [review]
This is awesome, just the one question.

File: docker-compose/docker-compose.yml (hunk 1 above)
Comment (rudigiesler) [reply docker-compose/docker-compose.yml]: Kind of. We could change this, but this means that I just need to create a token for one application. Let me take a look to see how difficult it is to have more databases in the postgres docker image

**Comment by Boxroma** [comment]
This is Great thanks!!

**Comment by rudigiesler** [comment]
@erikh360 Updated with separate DBs. Ready for review again

**Review by erikh360 — APPROVED** [review]

**Pull request merged**